kevm-pyk/src/tests/unit/test_kevm.py

```diff
@@ -50,3 +52,52 @@ def test_kinner_to_hex(test_id: str, input: KInner, result: KInner) -> None:
     to_hex = KEVM.kinner_to_hex(input)
     # Then
     assert to_hex == result
+
+
+JUMPDESTS_DATA: Final = [
```

Comment:
Since `BYTES` tokens are handled as a separate case, maybe it'd be safer to add a test for it? E.g.,

```
b"`\x80`@R4\x80\x15a\x00\x10W`\x00\x80\xfd[P`@Qa\x01a8\x03\x80a\x01a\x839\x81\x01`@\x81\x90Ra\x00/\x91a\x00KV[Pa\x01\x14V[cNH{q`\xe0\x1b`\x00R`A`\x04R`$`\x00\xfd[`\x00` \x80\x83\x85\x03\x12\x15a\x00^W`\x00\x80\xfd[\x82Q`\x01`\x01`@\x1b\x03\x80\x82\x11\x15a\x00uW`\x00\x80\xfd[\x81\x85\x01\x91P\x85`\x1f\x83\x01\x12a\x00\x89W`\x00\x80\xfd[\x81Q\x81\x81\x11\x15a\x00\x9bWa\x00\x9ba\x005V[`@Q`\x1f\x82\x01`\x1f\x19\x90\x81\x16`?\x01\x16\x81\x01\x90\x83\x82\x11\x81\x83\x10\x17\x15a\x00\xc3Wa\x00\xc3a\x005V[\x81`@R\x82\x81R\x88\x86\x84\x87\x01\x01\x11\x15a\x00\xdbW`\x00\x80\xfd[`\x00\x93P[\x82\x84\x10\x15a\x00\xfdW\x84\x84\x01\x86\x01Q\x81\x85\x01\x87\x01R\x92\x85\x01\x92a\x00\xe0V[`\x00\x86\x84\x83\x01\x01R\x80\x96PPPPPPP\x92\x91PPV[`?\x80a\x01\"`\x009`\x00\xf3\xfe`\x80`@R`\x00\x80\xfd\xfe\xa2dipfsX\"\x12 \xce\x0fs\x04\xc1\x8d\x10\xc8\xefe\x12(+\xa0\xff(\xa2\x95W\xa4\xeb\r\xc2\xf0\xbb((\x83\xfe\xed\t dsolcC\x00\x08\x19\x003\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00@" +Bytes VV1_b_114b9705:Bytes
```

That corresponds to deploying 
```solidity
contract Constr {
    constructor(bytes memory b) {  }
}
```

Reply:
After a discussion with Andrei, I take this comment back — `bytes` constructor arguments aren't supported and are problematic because they have symbolic length and, therefore, offsets. They don't seem common enough to get it addressed now.